src/curate_gpt/cli.py

```diff
@@ -1683,6 +1685,76 @@ def _text_lookup(obj: Dict):
     db.update_collection_metadata(collection, object_type="OntologyClass")
 
 
+@main.group()
+def embeddings():
+    """Command group for handling embeddings."""
+    pass
+
+
+def download_file(url):
+    """
+    Helper function to download a file from a URL to a temporary file.
+    """
+    local_filename = tempfile.mktemp()
+    with requests.get(url, stream=True) as r:
+        r.raise_for_status()
+        with open(local_filename, 'wb') as f:
+            for chunk in r.iter_content(chunk_size=8192):
+                f.write(chunk)
+    return local_filename
+
+
+def load_embeddings(file_path, embedding_format=None):
+    """
+    Helper function to load embeddings from a file. Supports Parquet and CSV formats.
+    """
+    if file_path.endswith('.parquet') or file_path.endswith('.parquet.gz') or embedding_format == 'parquet':
+        df = pd.read_parquet(file_path)
+    elif file_path.endswith('.csv') or file_path.endswith('.csv.gz') or embedding_format == 'csv':
+        df = pd.read_csv(file_path)
+    else:
+        raise ValueError(
+            "Unsupported file type. Only Parquet and CSV files are supported.")
+    return df.to_dict(orient='records')
+
+
+@embeddings.command(name="index")
+@path_option
+@collection_option
+@model_option
+@append_option
+@click.option("--embedding-format", "-f",
+              type=click.Choice(['parquet', 'csv']), help="Format of the input file")
+@click.argument("file_or_url")
+def index(path, collection, append, embedding_format, model, file_or_url):
```

Comment:
We already have `index` as a command, which is maybe a bit of a misnomer, as it actuallly embeds and indexes.

I know this command is a subcommand of `embeddings` but I think it's good to have consistent terminology... happy to change the name of the existing command (something like `load` may be better...)

Reply:
@cmungall oh okay, I can refactor

I did see the `index` command, and started putting my code there, but as I understood it this creates embeddings for a file and then loads those embeddings into chromadb. My code expects the embeddings directly, not a file that is to be embedded.  (please correct me if I misunderstand) 

should I put this code in `index` with a flag like `--embeddings_file` or something, so as to skip the indexing bit?

Reply:
(previous comment deleted - I misunderstood)

Reply:
> We already have index as a command, which is maybe a bit of a misnomer, as it actuallly embeds and indexes.

Agree - I changed this function to `load()`

> I know this command is a subcommand of embeddings but I think it's good to have consistent terminology... happy to change the name of the existing command (something like load may be better...)

I think `index` is a pretty good name for the `ontology index` subcommand, since it is not just loading